pymeasure/instruments/lakeshore/lakeshore331.py

```diff
@@ -56,8 +53,13 @@ class LakeShore331(SCPIUnknownMixin, Instrument):
 
     output_2 = Instrument.ChannelCreator(LakeShoreHeaterChannel, 2)
 
-    def __init__(self, adapter, name="Lakeshore Model 336 Temperature Controller", **kwargs):
-        kwargs.setdefault('read_termination', "\r\n")
+    def __init__(self,
+                 adapter,
+                 name="Lakeshore Model 336 Temperature Controller",
+                 read_termination="\r\n",
+                 **kwargs):
+        kwargs.setdefault('read_termination', read_termination)
+        warnings.warn("LakeShore331 is deprecated. Use LakeShore3xx instead.", DeprecationWarning)
```

Comment:
Please put also a note to the docstrings of the class. See https://github.com/pymeasure/pymeasure/issues/672#issuecomment-1371900222